.eslintrc.js

```diff
@@ -1,28 +1,58 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
   extends: [
+    'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
-    'prettier/@typescript-eslint',
     'plugin:prettier/recommended',
     'plugin:react/recommended',
+    'plugin:cypress/recommended',
```

Comment:
CI nám padá na typech pro testy (odkaz na [log](https://github.com/cesko-digital/web/pull/106/checks?check_run_id=1546881834)). Za mne dobrý nápad s využitím pluginů pro Cypress a Jest, mohu jen poprosit o správné nakonfigurování, tak aby se nám nehádali mezi sebou a fungovali tak, jak mají?